Descriptors/StaffClassificationDescriptor.xml

```diff
@@ -18,8 +18,7 @@
 		<Description>Teachers of general level instruction and/or services delivery classified by state and local practice from birth to Kindergarten.</Description>
 		<Namespace>uri://ed-fi.org/StaffClassificationDescriptor</Namespace>
 	</StaffClassificationDescriptor>
-	</StaffClassificationDescriptor>
-		<StaffClassificationDescriptor>
+	<StaffClassificationDescriptor>
```

Comment:
@mikeholm-simpat @mragonech 

We need to figure out how to prevent this from occurring in the first place. The mistake itself is not such a big deal - we all make mistakes from time to time. But this is a mistake that is easy to detect with a small amount of automation, and I thought we had that automation. I thought wrong.